src/main/bin/ccm-purge

```diff
@@ -116,14 +115,15 @@ sub init
                'help'      => \$help,
         ) or die("error processing command line: $!\n $usage");
 
-    die("Usage: $usage\n") if ($help || !$TimeAging);
-
     if (!$ccm_conf) {
         initCfg();
     } else {
         initCfg($ccm_conf);
     }
 
+    $TimeAging = EDG::WP4::CCM::CCfg::getCfgValue("purge_time") if (!$TimeAging);
```

Comment:
Thanks @stdweird - I suppose a value of zero could make sense, if you wanted to use purge_time alone. I also took the if (!$TimeAging) check on the getCfgValue, since I don't think this should ever be undefined.